src/main/frontend/state.cljs

```diff
@@ -275,7 +275,7 @@
      :graph/importing-state                 {}
 
      :whiteboard/onboarding-whiteboard?     (or (storage/get :ls-onboarding-whiteboard?) false)
-     })))
+     :whiteboard/onboarding?                (or (storage/get :whiteboard-onboarding?) false)})))
```

Comment:
The two keys is a bit confusing. Shall we merge them into one mapping, like `file-sync/onboarding-state`?

Reply:
That's what I did originally, but `storage/set` and `storage/set` remove the namespace from the key by using `(name key)`. The local storage key will be `onboarding-state?`, which could lead to ambiguities and hard to debug errors. `:ls-onboarding-whiteboard?` is already out, so I didn't want to change that.

I do agree that `:whiteboard-onboarding?` is not the best choice though. How about `:whiteboard-onboarding-state?`?

Reply:
`tour` sounds perfect, let's go with that!

Reply:
Renamed to `:whiteboard-onboarding-tour?`